src/main/java/edition/academy/seventh/repositories/BooksRepository.java

```diff
@@ -0,0 +1,41 @@
+package edition.academy.seventh.repositories;
+
+import edition.academy.seventh.database.connectors.EntityConnector;
+import edition.academy.seventh.database.connectors.H2Connector;
+import edition.academy.seventh.database.connectors.PostgresConnector;
+import org.springframework.stereotype.Repository;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityTransaction;
+import edition.academy.seventh.database.models.Book;
+import java.util.List;
+
+/**
+ * Repository to persist books in PostgreSQL database.
+ *
+ * @author Agnieszka Trzewik
+ */
+@Repository
+public class BooksRepository {
+
```

Comment:
Proponuje wygenerować sobie raporty checkStyle'a. Dużo ostrzeżeń apropo java doc'a i "indetation level" aczkolwiek nie wiem co to jest więc to puszczę.